taqueria-plugin-taquito/instantiate_account.ts

```diff
@@ -0,0 +1,42 @@
+import { getCurrentEnvironmentConfig, sendAsyncErr, sendJsonRes, sendWarn } from '@taqueria/node-sdk';
+import {
+	generateAccountKeys,
+	getDeclaredAccounts,
+	getNetworkInstantiatedAccounts,
+	getNetworkWithChecks,
+	InstantiateAccountOpts as Opts,
+} from './common';
+
+const instantiate_account = async (parsedArgs: Opts): Promise<void> => {
+	const env = getCurrentEnvironmentConfig(parsedArgs);
+	if (!env) return sendAsyncErr(`There is no environment called ${parsedArgs.env} in your config.json.`);
+	try {
+		const networkConfig = await getNetworkWithChecks(parsedArgs, env);
+		const declaredAccountNames = Object.keys(getDeclaredAccounts(parsedArgs));
+		const instantiatedAccounts = getNetworkInstantiatedAccounts(networkConfig);
+		let accountsInstantiated = [];
```

Comment:
Could we do a map or reduce here instead rather than have mutable variables here? Easier to reason about.

Reply:
I tried. Ran into concurrency issues so I rewrote it with loops